### ISSUE TITLE
Add Westend and/or custom endpoint support

### ISSUE THREAD

**Issue by chevdor** [polkawallet-io/polkawallet-flutter#204]
Adding Westend would be nice to offer a risk free dev environment.

**Comment by RomeroYang** [comment]
> Adding Westend would be nice to offer a risk free dev environment.

we are rebuilding the polkwallet app and this repo will be archived.

so if you want to run the app in dev, you'd better use the new repo https://github.com/polkawallet-io/app.

unfortunately the rebuilding is still ongoing and we do not have docs now, maybe you can wait for a few days.

**Comment by chevdor** [comment]
yes, no problem, just let me know once this is sorted. No stress :)

**Comment by RomeroYang** [comment]
> yes, no problem, just let me know once this is sorted. No stress :)

I added some descriptions here [README.md](https://github.com/polkawallet-io/app/blob/master/README.md), hope it can help.
